src/emmy/simplify.cljc

```diff
@@ -74,7 +74,7 @@
         expr
         (let [canonicalized-expr (canonicalize new-expr)]
           (cond (= canonicalized-expr expr) expr
-                (v/zero?
+                (g/numeric-zero?
```

Comment:
a change!!

Reply:
This is the one where what's being tested for zero is a lazy seq returned from simplify that got me wondering about this... I wanted to avoid the idea that a realized empty list could spoof this. I can put it back if you prefer

Reply:
BTW, I think what you had originally written is perfectly safe, in hindsight